packages/core/saved-objects/core-saved-objects-browser-internal/src/saved_objects_client.ts

```diff
@@ -306,6 +306,7 @@ export class SavedObjectsClient implements SavedObjectsClientContract {
       aggs: 'aggs',
       namespaces: 'namespaces',
       preference: 'preference',
+      downwardConversion: 'downwardConversion',
```

Comment:
I don't think we need to add this here given we did not add the option to the HTTP APIs?

Reply:
Removing it gives me an error:
```
Argument of type '{ defaultSearchOperator: string; fields: string; hasReference: string; hasReferenceOperator: string; hasNoReference: string; hasNoReferenceOperator: string; page: string; perPage: string; ... 7 more ...; preference: string; }' is not assignable to parameter of type 'Record<"type" | "page" | "perPage" | "sortField" | "fields" | "search" | "searchFields" | "hasReference" | "hasReferenceOperator" | "hasNoReference" | "hasNoReferenceOperator" | ... 5 more ... | "downwardConversion", string | ... 1 more ... | symbol>'.
  Property 'downwardConversion' is missing in type '{ defaultSearchOperator: string; fields: string; hasReference: string; hasReferenceOperator: string; hasNoReference: string; hasNoReferenceOperator: string; page: string; perPage: string; ... 7 more ...; preference: string; }' but required in type 'Record<"type" | "page" | "perPage" | "sortField" | "fields" | "search" | "searchFields" | "hasReference" | "hasReferenceOperator" | "hasNoReference" | "hasNoReferenceOperator" | ... 5 more ... | "downwardConversion", string | ... 1 more ... | symbol>'
```